plugins/render/api_key_test.go

```diff
@@ -0,0 +1,55 @@
+package render
```

Comment:
Looks like this file has some TODOs left, please have a look over them 😄

Reply:
I have update it.

Reply:
> Looks like this file has some TODOs left, please have a look over them 😄

I have updated it.